script/worker

```diff
@@ -30,6 +30,16 @@ specify the public key needed for API authentication
 
 specify the secret key needed for API authentication
 
+=item B<--token> <value>
+
+specify a comma-separated list of tokens for C<host>@C<key>@C<secret>
+combinations as alternative to the individual options. Alternatively use the
+environment variable C<OPENQA_WORKER_TOKEN>.
+
+=item B<--encode-token> HOST@KEY@SECRET
```

Comment:
I think the advantage of using a token is also secure the `secret`. wouldnt make it more "secure" if you do not have to expose the secret in the terminal?

Reply:
yea thats why I quote the term `secure`. thanks for reply
I will resolve the thread and I see your point but I would like to bring the topic up again in the future.